tests/test-helper.js

```diff
@@ -1,11 +1,11 @@
-import resolver from './helpers/resolver';
-import {
-  setResolver
-} from 'ember-qunit';
-import { start } from 'ember-cli-qunit';
+import Application from '../app';
+import { setApplication } from '@ember/test-helpers';
+import { start } from 'ember-qunit';
 import './helpers/flash-message';
 import loadEmberExam from 'ember-exam/test-support/load';
 
 loadEmberExam();
-setResolver(resolver);
+
+setApplication(Application.create({ autoboot: false }));
+
 start();
```

Comment:
Simplified, but nothing custom here. Just took the new version from 

https://github.com/ember-cli/ember-new-output/blob/v2.17.0/tests/test-helper.js